<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoviesGUI_Display</class>
 <widget class="QDialog" name="MoviesGUI_Display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonBack_themovie">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>545</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle_themovie">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>750</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight SemiConde</family>
     <pointsize>45</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Movie Title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDirector_themovie">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight SemiConde</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Director:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDate_themovie">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight SemiConde</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Release Date: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLength_themovie">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight SemiConde</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Movie Length: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGenre_themovie">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>320</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight SemiConde</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Genre: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPic_themovie">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>200</width>
     <height>296</height>
    </rect>
   </property>
   <property name="text">
    <string>POSTER</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRating_themovie">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonRating_themovie">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>440</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit Rating</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRating_themovie">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiLight SemiConde</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Average Rating: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
